### PR TITLE
OSDOCS-7679: Improve text in VPC section

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-7679](https://issues.redhat.com//browse/OSDOCS-7679): Improve text in the ROSA shared VPC section

Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7679

Link to docs preview:
ROSA: https://64502--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-shared-vpc-config

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-sharing-vpc-cluster-creation.adoc [hunk 1]
```diff
@@ -3,25 +3,24 @@
...
 :_content-type: PROCEDURE
 [id="rosa-sharing-vpc-cluster-creation_{context}"]
-= Creating your cluster in a shared VPC
+= Step Four - Cluster Creator: Creating your cluster in a shared VPC
 To create a cluster in a shared VPC, complete the following step. 
 
 [NOTE]
 ====
-Installing a cluster in a shared VPC is supported only for OpenShift 4.13.9 and later.
+Installing a cluster in a shared VPC is supported only for OpenShift 4.13.10 and later.
```
Comment (wgordon17) [inline modules/rosa-sharing-vpc-cluster-creation.adoc:11]:
```suggestion
Installing a cluster in a shared VPC is supported only for OpenShift 4.12.34 and later 4.12.z's, and 4.13.10 and all future 4.13.z's and 4.y's.
```

We have the 4.12 backports in place, starting at 4.12.34. But I don't really have a great way to say
`4.12.34+` within 4.12 and `4.13.10+` for _all_ values of `+` 😉

Comment (wgordon17) [reply modules/rosa-sharing-vpc-cluster-creation.adoc:11]: The logic is documented in https://issues.redhat.com/browse/OCM-3688

Also, why did we change `4.13.9` -> `4.13.10`? It looks like `4.13.9` was the _correct_ minimum 4.13 version

File: rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
+.Prerequisites for the *Cluster Creator*
+* You installed the ROSA CLI (`rosa`) 1.2.26 or later.
+* You created all of the required ROSA account roles for creating a cluster.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
+* Both AWS accounts belong to the same AWS organization.
+
 [NOTE]
 ====
-Installing a cluster in a shared VPC is supported only for OpenShift 4.13.9 and later.
+Installing a cluster in a shared VPC is supported only for OpenShift 4.13.10 and later.
```
Comment (wgordon17) [inline rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc:34]: Same comment from above https://github.com/openshift/openshift-docs/pull/64502/files#r1325986488

File: rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc [hunk 3]
```diff
@@ -1,24 +1,37 @@
...
 * You have an AWS account with the proper permissions to create roles and share resources.
-* You are using an AWS account to create your cluster ("*cluster-creating AWS account*") that is separate from the AWS account that creates your VPC ("*VPC-owning AWS account*").
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
 * Both AWS accounts belong to the same AWS organization.
 * You enabled resource sharing from the management account for your organization.
 * You have access to the link:https://signin.aws.amazon.com[AWS console].
 
+.Prerequisites for the *Cluster Creator*
+* You installed the ROSA CLI (`rosa`) 1.2.26 or later.
+* You created all of the required ROSA account roles for creating a cluster.
```
Comment (wgordon17) [inline rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc:28]: Considering this is an assembly, can we link to https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html#rosa-sts-creating-account-wide-sts-roles-and-policies_rosa-sts-creating-a-cluster-quickly (or something similar) in order to explain _how_ to create the Account Roles?

Comment (jneczypor) [reply rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc]: No problem!

File: rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc [hunk 4]
```diff
@@ -1,24 +1,37 @@
...
+.Prerequisites for the *VPC Owner*
 * You have an AWS account with the proper permissions to create roles and share resources.
-* You are using an AWS account to create your cluster ("*cluster-creating AWS account*") that is separate from the AWS account that creates your VPC ("*VPC-owning AWS account*").
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
 * Both AWS accounts belong to the same AWS organization.
 * You enabled resource sharing from the management account for your organization.
 * You have access to the link:https://signin.aws.amazon.com[AWS console].
 
+.Prerequisites for the *Cluster Creator*
+* You installed the ROSA CLI (`rosa`) 1.2.26 or later.
```
Comment (wgordon17) [inline rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc:27]:
```suggestion
* You installed the link:https://console.redhat.com/openshift/downloads#tool-rosa[ROSA CLI (`rosa`)] 1.2.26 or later.
```

File: modules/rosa-sharing-vpc-creation-and-sharing.adoc [hunk 5]
```diff
@@ -100,9 +100,9 @@ $ aws iam attach-role-policy --role-name <role_name> --policy-arn \  <1>
...
 <1> Replace _<role_name>_ with the name of the role you created.
-<2> Replace _<AWS_account_ID>_ with the *VPC-owning AWS account* ID.
+<2> Replace _<AWS_account_ID>_ with the *VPC Owner's* AWS account ID.
 --
 +
-. In the link:https://us-east-1.console.aws.amazon.com/ram/[Resource Access Manager of the AWS console], create a Resource Share that shares the previously created public and private subnets to the *cluster-creating AWS account* ID.
+. In the link:https://us-east-1.console.aws.amazon.com/ram/[Resource Access Manager of the AWS console], create a resource share that shares the previously created public and private subnets with the *Cluster Creator's* AWS account ID.
 
-After you create the Resource Share, notify the *cluster-creating AWS account* user to reserve an `openshiftapps.com` DNS domain and create Operator roles to continue configuration.
+. After you create the resource share, share the `SharedVPCRole` ARN with the *Cluster Creator* to continue configuration.
```
Comment (wgordon17) [inline modules/rosa-sharing-vpc-creation-and-sharing.adoc:108]:
```suggestion
. After you create the resource share, provide the `SharedVPCRole` ARN to the *Cluster Creator* to continue configuration.
```

File: modules/rosa-sharing-vpc-dns-and-roles.adoc [hunk 6]
```diff
@@ -3,17 +3,17 @@
 // * networking/rosa-shared-vpc-config.adoc
 :_content-type: PROCEDURE
 [id="rosa-sharing-vpc-dns-and-roles_{context}"]
-= Creating your DNS and cluster creation roles
+= Step Two - Cluster Creator: Creating your DNS and cluster creation roles
```
Comment (wgordon17) [inline modules/rosa-sharing-vpc-dns-and-roles.adoc:6]:
```suggestion
= Step Two - Cluster Creator: Reserving your DNS and creating cluster operator roles
```

File: modules/rosa-sharing-vpc-dns-and-roles.adoc [hunk 7]
```diff
@@ -63,10 +66,11 @@ $ rosa create operator-roles --oidc-config-id <oidc-config-ID> <1>
 The Installer account role and the shared VPC role must have a one-to-one relationship. If you want to create multiple shared VPC roles, you should create one set of account roles per shared VPC role.
 ====
 
-After you create the Operator roles, share the full domain name, which is created with `<intended_cluster_name>.<created_dns_domain>`, your _Ingress Operator Cloud Credentials_ and _Installer_ roles' ARN with the *VPC-owning AWS account* user.
-
-This information resembles these examples:
-
+. After you create the Operator roles, share the full domain name, which is created with `<intended_cluster_name>.<created_dns_domain>`, your _Ingress Operator Cloud Credentials_ role's ARN, and your _Installer_ role's ARN with the *VPC Owner* to continue configuration.
```
Comment (wgordon17) [inline modules/rosa-sharing-vpc-dns-and-roles.adoc:69]:
```suggestion
. After you create the Operator roles, share the full domain name, which is created with `<intended_cluster_name>.<reserved_dns_domain>`, your _Ingress Operator Cloud Credentials_ role's ARN, and your _Installer_ role's ARN with the *VPC Owner* to continue configuration.
```

**Review by wgordon17 — DISMISSED** [review]
Excellent! Looks great 🤩 Just these quick suggestions, and we'll be good to go once we get the diagrams 👍

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**